app/Services/TssRevisionService.php

```
@@ -0,0 +1,31 @@
+                'boms',
+                'attachments'
+            ]);
+            $version = Revision::where('project_id', $project->id)->max('version') + 1;
+            Revision::create([
+                'project_id' => $project->id,
+                'project_uuid' => $project->uuid,
+                'data' => json_encode(ProjectTssRevisionResource::make($project)->toArray($request)),
+                'comments' => $request->input('comments'),
+                'version' => $version,
+            ]);
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'createTssRevision'. (undefined)

(StaticAccess)